airmozilla/main/views.py

```diff
@@ -482,6 +485,45 @@ def get(self, request, slug):
 
         return render(request, self.template_name, context)
 
+
+def es_to_template(self, request, slug):
+    es = pyelasticsearch.ElasticSearch('http://localhost:9200/')
+    for event in Event.objects.scheduled().order_by('?')[:100]:
+        # should do bulk ops really but besides the point
+        es.index(
+            'events',
+            'event',
+            {
+                'title': event.title,
+                'tags': [x.name for x in event.tags.all()],
+                'channels': [x.name for x in event.channels.all()],
+            },
+            id=event.id,
+        )
+
+    es.refresh()  # let it clear its throat
+    print '-' * 100
+
+    # get one my id
+    event, = Event.objects.scheduled().order_by('?')[:1]
+    try:
+        doc = es.get('events', 'event', event.id)
+        pprint(doc['_source'])
+        assert doc['_source']['title'] == event.title
+    except pyelasticsearch.exceptions.ElasticHttpNotFoundError:
+        print "Not indexed yet :("
+
+    print '-' * 100
+    hits = es.search('title: firefox', index='events')['hits']
+    for doc in hits['hits']:
+        event = Event.objects.get(id=doc['_id'])
+        print "\t", repr(event.title), doc['_score']
+   
+    template_name = 'main/es.html'
+    ids = [int(x) for x in id.split(',')]
+    events = Event.objects.filter(id__in=ids)
+    return render(request, template_name, events)
```

Comment:
Independent of what goes above this line, this line needs to be:
```python
return render(request, 'main/es.html', context)
```
And the line above should be:
```python
context = {
    'events': events,
}
```
If you do that, you're going to make big progress here.